### PR TITLE
feat: add Azure B2C to Social Login Keys [CU-1rww0rp]

### PR DESCRIPTION
**Pull request by Alchez**

### CODE REVIEW

**Comment by vjFaLk** [comment]
Task linked: [CU-1rww0rp Figure out Azure's AD (Active Directory) B2C login for Avask](https://app.clickup.com/t/1rww0rp)

**Comment by Alchez** [comment]
PR made upstream to [develop](https://github.com/frappe/frappe/pull/15442)

**Comment by vjFaLk** [comment]
@Alchez how do I review this? It's really hard to differentiate formatting changes with actual ones...

**Comment by Alchez** [comment]
@vjFaLk, ignore the test files. They're files I've changed in the upstream PR. Everything else are the actual changes.

File: frappe/integrations/doctype/social_login_key/social_login_key.js [hunk 1]
```diff
@@ -69,10 +73,24 @@ frappe.ui.form.on('Social Login Key', {
...
 	clear_fields(frm) {
-		for(var field of fields){
+		for (const field of fields) {
 			frm.set_value(field, "");
 			frm.set_df_property(field, "read_only", 0);
 		}
-	}
+	},
 
+	azure_tenant_id(frm) {
```
Comment (vjFaLk) [inline frappe/integrations/doctype/social_login_key/social_login_key.js:82]: Add comment that mentions that this is a fieldname trigger

**Pull request merged**